.eslintrc.js

```diff
@@ -262,17 +262,19 @@ module.exports = {
   'overrides': [
     {
       'files': [
-        'test/**/*.js',
-        'extensions/**/test/**/*.js',
-        'extensions/**/test-e2e/*.js',
         'ads/**/test/**/*.js',
+        'examples/visual-tests/**/*.js',
+        'extensions/**/test-e2e/*.js',
+        'extensions/**/test/**/*.js',
+        'test/**/*.js',
         'testing/**/*.js',
       ],
       'rules': {
         'require-jsdoc': 0,
         'local/always-call-chai-methods': 2,
         'local/no-bigint': 0,
         'local/no-dynamic-import': 0,
+        'local/no-for-of-statement': 0,
```

Comment:
I think this top-level `.eslintrc.js` can now be fully reverted.

Reply:
Based on your comment about `for...of`, all that's left to confirm is whether `'examples/visual-tests/**/*.js',` needs to be added to the `overrides` in the repo-root `.eslintrc.js`.

Reply:
Reverted since it's not needed for this change. But I still want to use `for..of` in tests (#32396 😇 )

Reply:
Agree, and thank you for splitting off this change. Lint warnings are particularly annoying because they are non-blocking by design, and can sneak up on you when you're not looking.

Can confirm this PR is clean: https://app.circleci.com/pipelines/github/ampproject/amphtml/1102/workflows/9376733f-5d51-46e2-82da-f0d5560fa14d/jobs/9931

![image](https://user-images.githubusercontent.com/26553114/106795962-bf59f200-6628-11eb-8382-521d550914e8.png)